### ISSUE TITLE
Allow subsonic browse by folder structure

### ISSUE THREAD

**Issue by thegr8brian** [clementine-player/Clementine#4251]
I'd like to request that browsing of a subsonic server be made possible by the folder structure instead of only tags.  

The use case I'm running in to is that when multiple codec types(FLAC,MP3,etc) of an album exist on a subsonic server, browsing by tags currently makes it impossible to distinguish between them.  

See here for API: http://www.subsonic.org/pages/api.jsp#getMusicFolders

**Comment by alexfornuto** [comment]
:+1:  for the feature request. The most obvious reason why is for compilation albums, soundtracks, or other albums with multiple various artists. Here's a screenshot with the album "Nightmare Revisited" as example:

![Example](http://imgur.com/ZQnV1Fz)

**Comment by alanbriolat** [comment]
@alexfornuto This is what the "album artist" tag is for (assuming your music is tagged correctly).
![image](https://cloud.githubusercontent.com/assets/12193/12867440/9b92bbae-cce5-11e5-958f-5173625e9cda.png)

![image](https://cloud.githubusercontent.com/assets/12193/12867433/7e3c3954-cce5-11e5-896e-b47063656d3d.png)

**Comment by alexfornuto** [comment]
Ahh, thanks! I'd still like this feature, for other groupings I make not based on artist or albums.

**Comment by unnilennium** [comment]
this would be a great improvement to access multiple subset of collection. Subsonic and Madsonic allow to browse different folders for different collections. 
I have as an example : 
- a folder with soundtracks : album artist  won't really do 
- a folder with music by artist / album; album artist is good.
- a folder with only compilations: album artist does not do fine neither.

and again It is a nightmare without at least  folder collection support if I want to shuffle only on soundtracks ( and not soundtrack genre will not do, as there might have some classical or other genre in soundtracks)

**Comment by eharris** [comment]
I agree.  In my opinion, only being able to browse by artist is unusable if you have a large library.  My library has nearly 10,000 artists and it's very hard to find stuff in a way that is logical to me without folder browsing.

**Comment by jaimelive** [comment]
Pretty please? I always end up having to use the browser instead of clementine :(

**Comment by l1m1t0** [comment]
I just installed the application, disappointed there is no folder structure grouping.

**Comment by JuakinBM** [comment]
Please?

**Comment by alexfornuto** [comment]
It's been twelve years, I doubt it.
